packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1126,7 +1130,8 @@ export class LitNodeClientNodeJs extends LitCore {
     log("minimum required length:", this.config.minNodeCount);
 
     if (validatedSignedData.length < this.config.minNodeCount) {
-      throw new Error(`not enough nodes to get the signatures.  Expected ${this.config.minNodeCount}, got ${validatedSignedData.length}`);
+      log(`not enough nodes to get the signatures.  Expected ${this.config.minNodeCount}, got ${validatedSignedData.length}`);
```

Comment:
@joshLong145 so that it could continue this line https://github.com/LIT-Protocol/js-sdk/blob/9a0d7bb7ccdb2f9347879b4b733b7f25362d7e84/packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts#L1394-L1420